test/api/CLI.test.js

```diff
@@ -8,19 +8,17 @@ describe("CLI API", () => {
     });
 
     describe("makeCommand", () => {
-        it("should make command", async (done) => {
+        it("should make command", async () => {
```

Comment:
I think we should return `Promise` here based on https://jestjs.io/blog/2021/05/25/jest-27#features-coming-with-breaking-changes

Reply:
I think we already return a void promise here as per the error - 

<img width="969" alt="Screenshot 2021-06-02 at 6 08 02 PM" src="https://user-images.githubusercontent.com/46647141/120481266-8f9bda80-c3cd-11eb-8b34-8bff31fe7275.png">

Reply:
We should return https://github.com/webpack/webpack-cli/blob/03c5030537dd47e90abbca1af133f7d8c8176a8d/test/api/CLI.test.js#L18, because now we nothing return, other solution is using https://jestjs.io/ru/docs/expect#expectassertionsnumber

Reply:
Updated with `expect.assertions()`